### PR TITLE
add Chinese models in LLM chatbot

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
**Done:**
- QWEN
- ChatGLM2

**Attempt to add:**
- Baichuan2 (failed, version conflict between Transformers and Optimum-intel )

### CODE REVIEW

**Comment by OpenVINO-dev-contest** [comment]
@eaidova

**Comment by OpenVINO-dev-contest** [comment]
@zhuo-yoyowz @raymondlo84 maybe you can also help to test : )

File: notebooks/254-llm-chatbot/254-llm-chatbot.ipynb [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```
Comment (eaidova) [inline notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: <p>&gt;&gt;please refer to the <a href="https://github.com/QwenLM/Qwen" target="_blank">GitHub</a> code repositoryel.</p><p> </p><p>not sure it is some formatting in editor issue or typo</p><p> </p><p>BTW, maybe chatglm3-6b also can be added?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (eaidova) [inline notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: <pre class="ql-syntax" spellcheck="false">Line #21.        if model_configuration["remote"] is False:
</pre><p>if not model_configuration["remote"]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (eaidova) [inline notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: <pre class="ql-syntax" spellcheck="false">Line #68.                        history_template.format(num=round, user=item[0], assistant=item[1])
</pre><p>does this changes compatible with models where history_template does not contains round?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: no, it also works for qwen which does not includes round

Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: chatglm3 has more complicated input template, which is not compatible with our current template.

Comment (eaidova) [inline notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: <pre class="ql-syntax" spellcheck="false">Line #1.    from functools import wraps
</pre><p>with increasing number of supported models this code section become more and more large, I think maybe it is time for moving it in separated file?  it also will allow do not store model to converter mapping in notebook and directly map on configuration level</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by eaidova** [comment]
@OpenVINO-dev-contest thank you for the great addition for notebook!

spell checker run: https://github.com/openvinotoolkit/openvino_notebooks/actions/runs/6858205118/job/18648585178?pr=1448
failed due to added some unknown words,  could you please extend the exception list for them in this file: https://github.com/openvinotoolkit/openvino_notebooks/blob/main/.ci/spellcheck/.pyspelling.wordlist.txt?

File: notebooks/254-llm-chatbot/254-llm-chatbot.ipynb (hunk 1 above)
Comment (OpenVINO-dev-contest) [reply notebooks/254-llm-chatbot/254-llm-chatbot.ipynb:61]: done

**Review by eaidova — APPROVED** [review]

**Pull request merged**